Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
Use @@note instead. For example

```
@@note ""Try not to care"". ...
```

This generates a begin/end block in LaTeX and a div in HTML.